(URGENT) Relaton `concatenate` crashing due to namespace issues

This occurs in https://github.com/metanorma/mn-samples-iec/ after running `bundle update`.

```sh
echo "Running locally..."; bundle exec relaton concatenate \
	  -t "Metanorma-IEC sample documents" \
		-g "International Electrotechnical Commission" \
		documents documents.rxl
Running locally...
echo "Running locally..."; bundle exec relaton xml2html documents.rxl
Running locally...
bundler: failed to load command: relaton (/Users/me/.rbenv/versions/2.6.5/bin/relaton)
Nokogiri::XML::XPath::SyntaxError: ERROR: Undefined namespace prefix: ./xmlns:relaton-collection/xmlns:title
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/nokogiri-1.10.7/lib/nokogiri/xml/searchable.rb:198:in `evaluate'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/nokogiri-1.10.7/lib/nokogiri/xml/searchable.rb:198:in `xpath_impl'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/nokogiri-1.10.7/lib/nokogiri/xml/searchable.rb:179:in `xpath_internal'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/nokogiri-1.10.7/lib/nokogiri/xml/searchable.rb:154:in `xpath'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/nokogiri-1.10.7/lib/nokogiri/xml/searchable.rb:55:in `search'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/nokogiri-1.10.7/lib/nokogiri/xml/searchable.rb:67:in `at'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-cli-0.4.2/lib/relaton/element_finder.rb:10:in `find'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-cli-0.4.2/lib/relaton/element_finder.rb:6:in `find_text'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-cli-0.4.2/lib/relaton/bibcollection.rb:38:in `from_xml'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-cli-0.4.2/lib/relaton/cli/xml_to_html_renderer.rb:72:in `build_bibcollection'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-cli-0.4.2/lib/relaton/cli/xml_to_html_renderer.rb:44:in `build_liquid_document'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-cli-0.4.2/lib/relaton/cli/xml_to_html_renderer.rb:16:in `render'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-cli-0.4.2/lib/relaton/cli/xml_to_html_renderer.rb:32:in `render'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-cli-0.4.2/lib/relaton/cli/base_convertor.rb:55:in `convert_to_html'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-cli-0.4.2/lib/relaton/cli/base_convertor.rb:23:in `to_html'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-cli-0.4.2/lib/relaton/cli/base_convertor.rb:43:in `to_html'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-cli-0.4.2/lib/relaton/cli/command.rb:95:in `xml2html'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/base.rb:466:in `start'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-cli-0.4.2/lib/relaton/cli.rb:9:in `start'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-cli-0.4.2/exe/relaton:20:in `<top (required)>'
  /Users/me/.rbenv/versions/2.6.5/bin/relaton:23:in `load'
  /Users/me/.rbenv/versions/2.6.5/bin/relaton:23:in `<top (required)>'
make: *** [documents.html] Error 1
```

--- a/docs/README.adoc
+++ b/docs/README.adoc
@@ -1,35 +1,33 @@
 = Relaton CLI (relaton-cli): Relaton Command-line Interface
 
 image:https://img.shields.io/gem/v/relaton-cli.svg["Gem Version", link="https://rubygems.org/gems/relaton-cli"]
-image:https://img.shields.io/travis/metanorma/relaton-cli/master.svg["Build Status", link="https://travis-ci.org/metanorma/relaton-cli"]
-image:https://ci.appveyor.com/api/projects/status/fyy87flpig4q5o2u?svg=true["Appveyor Build Status", link="https://ci.appveyor.com/project/ribose/relaton-cli"]
+image:https://github.com/relaton/relaton-cli/workflows/ubuntu/badge.svg["Build Status", link="https://github.com/relaton/relaton-cli/actions?workflow=ubuntu"]
+image:https://github.com/relaton/relaton-cli/workflows/macos/badge.svg["Build Status", link="https://github.com/relaton/relaton-cli/actions?workflow=macos"]
+image:https://github.com/relaton/relaton-cli/workflows/windows/badge.svg["Build Status", link="https://github.com/relaton/relaton-cli/actions?workflow=windows"]
 image:https://codeclimate.com/github/metanorma/relaton-cli/badges/gpa.svg["Code Climate", link="https://codeclimate.com/github/metanorma/relaton-cli"]
 
 Documentation in development.
 
-Please refer to https://github.com/metanorma/relaton.
+Please refer to https://github.com/relaton/relaton.
 
-== Executables
+== Commands
 
-The following executables are included:
+The following commands are provided.
 
 === relaton concatenate
 
 [source,console]
 ----
-$ relaton concatenate Source-Directory Collection-File -t TITLE -g ORGANIZATION -n
+$ relaton concatenate Source-Directory Relaton-Collection-File -t TITLE -g ORGANIZATION -n
 ----
 
-Iterates through all the Relaton YAML files in `Source-Directory`, and converts them in-place into Relaton XML.
+Iterates through all the Relaton files (YAML and XML) in `Source-Directory`, and concatenates them into a Relaton Collection file. The extension of the Collection file can be set using the `Relaton-Collection-File` file name (i.e, if it uses an extension of `yaml`, a Relaton YAML file will be created; if `rxl`, a Relaton XML file will be created, or via the `-x [ext]` (or `--extension`) option.
 
-Iterates through all the Relaton XML files in `Source-Directory` (including those just converted from Relaton YAML),
-and outputs to `Collection-File` a `relaton-collection` document containing all the Relaton XML objects read in.
-
-For each Relaton XML file, if a document file with the same base name is identified (i.e. an XML, HTML, PDF or DOC
+For each Relaton input file in the `Source-Directory`, if a document file with the same base name is identified (i.e. an XML, HTML, PDF or DOC
 file), a link to that file is inserted.
 
 If the `TITLE` or `ORGANIZATION` options are given, they are added to the `Collection-File` output as the
-title and author of the `relaton-collection` document.
+title and author of the `Relaton-Collection-File` document.
 
 If the `-n` or `--new` option given then use the new Relaton YAML format https://github.com/relaton/relaton-bib/blob/master/docs/hash.adoc.
 
@@ -37,11 +35,11 @@ If the `-n` or `--new` option given then use the new Relaton YAML format https:/
 
 [source,console]
 ----
-$ relaton split Collection-File Relaton-File-Directory -x rxl -n
+$ relaton split Relaton-Collection-File Relaton-File-Directory -x rxl -n
 ----
 
 Splits a `Relaton-Collection-File` into multiple files in the `Relaton-File-Directory`, and it also
-suports an addional `extension` options to use different extension.
+suports an additional `-x` or `--extension` options to use different extension.
 
 If the `-n` or `--new` option given then use the new Relaton YAML format https://github.com/relaton/relaton-bib/blob/master/docs/hash.adoc.
 
@@ -217,9 +215,9 @@ Render a Relaton YAML file (`filename.yaml`) as an HTML file. The stylesheet and
 $ relaton yaml2xmlnew <filename.yaml> -o OUTPUT-DIRECTORY -x RELATON_EXTENSION -p PREFIX -r LIBRARY
 ----
 
-Convert a Relaton YAML file (`filename.yaml`) in a new format into a Relaton XML file (`filename.xml`). If the Relaton YAML file specifies multiple bibliograph items, and `OUTPUT-DIRECTORY` is nominated, also convert the file into a list of Relaton XML files for each entry, stored in that directory. The document identifier is used as the name of each Relaton XML file; the Relaton XML filename is suffixed with `RELATON_EXTENSION` (default `.rxl`) and prefixed with `PREFIX` (default empty). Any libraries that need to be required for the conversion are specified in `LIBRARY` as a space-delimited list.
+Convert a Relaton YAML file (`filename.yaml`) in a new format into a Relaton XML file (`filename.xml`). If the Relaton YAML file specifies multiple bibliographic items, and `OUTPUT-DIRECTORY` is nominated, also convert the file into a list of Relaton XML files for each entry, stored in that directory. The document identifier is used as the name of each Relaton XML file; the Relaton XML filename is suffixed with `RELATON_EXTENSION` (default `.rxl`) and prefixed with `PREFIX` (default empty). Any libraries that need to be required for the conversion are specified in `LIBRARY` as a space-delimited list.
 
-A Relaton Colleciton YAML file contains some initial metadata, and a list of metadata about each bibliographic entry:
+A Relaton Collection YAML file contains some initial metadata, and a list of metadata about each bibliographic entry:
 
 [source,yaml]
 ----
@@ -228,11 +226,11 @@ root:
   title: CalConnect Standards Registry
   items:
     - technical_committee: PUBLISH
-      docid: 
+      docid:
         type: CC
         id: CC 36000
       type: standard
-      title: 
+      title:
         type: main
         content: Standardization documents -- Vocabulary
       docstatus:
@@ -241,11 +239,11 @@ root:
         type: issued
         value:  2018-10-25
     - technical_committee: DATETIME
-      docid: 
+      docid:
         type: CC
         id: CC 34000
       type: standard
-      title: 
+      title:
         type: main
         content: Date and time -- Concepts and vocabulary
       docstatus:
@@ -255,7 +253,7 @@ root:
         value: 2018-10-25
 ----
 
-A Relaton YAML file describing an individual bibliographic entry is limited to metadata specific to that entry. Flavor gems have aditional fields. The following illustrates the common fields supported by all flavor gems (see link:https://github.com/relaton/relaton-bib/blob/master/docs/hash.adoc#yaml[Relaton YAML]):
+A Relaton YAML file describing an individual bibliographic entry is limited to metadata specific to that entry. Flavor gems have additional fields. The following illustrates the common fields supported by all flavor gems (see link:https://github.com/relaton/relaton-bib/blob/master/docs/hash.adoc#yaml[Relaton YAML]):
 
 [source,yaml]
 ----
@@ -471,3 +469,4 @@ $ relaton yaml2xmlnew <filename.yaml> [<stylesheet>] [<liquid-template-dir>]
 ----
 
 Render a Relaton YAML file in a new format (`filename.yaml`) as an HTML file. The `stylesheet` and `liquid-template-dir` directories are as for <<relaton xml2html,relaton xml2html>>.
+

--- a/lib/relaton/cli/command.rb
+++ b/lib/relaton/cli/command.rb
@@ -19,7 +19,7 @@ module Relaton
       end
 
       desc "extract Metanorma-XML-File / Directory Relaton-XML-Directory", "Extract Relaton XML from Metanorma XML file / directory"
-      option :extension, aliases: :x, desc: "File extension of Relaton XML files, defaults to 'rxl'"
+      option :extension, aliases: :x, default: "rxl", desc: "File extension of Relaton XML files, defaults to 'rxl'"
 
       def extract(source_dir, outdir)
         Relaton::Cli::RelatonFile.extract(source_dir, outdir, options)
@@ -29,13 +29,14 @@ module Relaton
       option :title, aliases: :t,  desc: "Title of resulting Relaton collection"
       option :organization, aliases: :g, desc: "Organization owner of Relaton collection"
       option :new, aliases: :n, type: :boolean, desc: "Use the new Relaton YAML format"
+      option :extension, aliases: :x, default: "rxl", desc: "File extension of destination Relaton file, defaults to 'rxl'"
 
       def concatenate(source_dir, outfile)
         Relaton::Cli::RelatonFile.concatenate(source_dir, outfile, options)
       end
 
       desc "split Relaton-Collection-File Relaton-XML-Directory", "Split a Relaton Collection into multiple files"
-      option :extension, aliases: :x, desc: "File extension of Relaton XML files, defaults to 'rxl'"
+      option :extension, aliases: :x, default: "rxl", desc: "File extension of Relaton XML files, defaults to 'rxl'"
       option :new, aliases: :n, type: :boolean, desc: "Use the new Relaton YAML format"
 
       def split(source, outdir)
@@ -43,7 +44,7 @@ module Relaton
       end
 
       desc "yaml2xml YAML", "Convert Relaton YAML into Relaton Collection XML or separate files"
-      option :extension, aliases: :x, desc: "File extension of Relaton XML files, defaults to 'rxl'"
+      option :extension, aliases: :x, default: "rxl", desc: "File extension of Relaton XML files, defaults to 'rxl'"
       option :prefix, aliases: :p, desc: "Filename prefix of individual Relaton XML files, defaults to empty"
       option :outdir, aliases: :o,  desc: "Output to the specified directory with individual Relaton Bibdata XML files"
       option :require, aliases: :r, type: :array, desc: "Require LIBRARY prior to execution"
@@ -54,7 +55,7 @@ module Relaton
       end
 
       desc "yaml2xmlnew YAML", "Convert Relaton YAML into Relaton Collection XML or separate files"
-      option :extension, aliases: :x, desc: "File extension of Relaton XML files, defaults to 'rxl'"
+      option :extension, aliases: :x, default: "rxl", desc: "File extension of Relaton XML files, defaults to 'rxl'"
       option :prefix, aliases: :p, desc: "Filename prefix of individual Relaton XML files, defaults to empty"
       option :outdir, aliases: :o,  desc: "Output to the specified directory with individual Relaton Bibdata XML files"
       option :require, aliases: :r, type: :array, desc: "Require LIBRARY prior to execution"
@@ -65,7 +66,7 @@ module Relaton
       end
 
       desc "xml2yaml XML", "Convert Relaton XML into Relaton Bibdata / Bibcollection YAML (and separate files)"
-      option :extension, aliases: :x, desc: "File extension of Relaton YAML files, defaults to 'yaml'"
+      option :extension, aliases: :x, default: "yaml", desc: "File extension of Relaton YAML files, defaults to 'yaml'"
       option :prefix, aliases: :p, desc: "Filename prefix of Relaton XML files, defaults to empty"
       option :outdir, aliases: :o, desc: "Output to the specified directory with individual Relaton Bibdata YAML files"
       option :require, aliases: :r, type: :array, desc: "Require LIBRARY prior to execution"
@@ -76,7 +77,7 @@ module Relaton
       end
 
       desc "xml2yamlnew XML", "Convert Relaton XML into Relaton Bibdata / Bibcollection YAML (and separate files)"
-      option :extension, aliases: :x, desc: "File extension of Relaton YAML files, defaults to 'yaml'"
+      option :extension, aliases: :x, default: "yaml", desc: "File extension of Relaton YAML files, defaults to 'yaml'"
       option :prefix, aliases: :p, desc: "Filename prefix of Relaton XML files, defaults to empty"
       option :outdir, aliases: :o, desc: "Output to the specified directory with individual Relaton Bibdata YAML files"
       option :require, aliases: :r, type: :array, desc: "Require LIBRARY prior to execution"

--- a/lib/relaton/cli/relaton_file.rb
+++ b/lib/relaton/cli/relaton_file.rb
@@ -16,7 +16,7 @@ module Relaton
       end
 
       def concatenate
-        write_to_file(bibcollection.to_yaml)
+        concatenate_and_write_to_files
       end
 
       def split
@@ -128,7 +128,7 @@ module Relaton
           bibdata = Relaton::Bibdata.from_xml(bib.root)
           build_bibdata_relaton(bibdata, file)
 
-          write_to_file(bibdata.to_xml, outdir, build_filename(file))
+          write_to_file(bibdata.send(output_type), outdir, build_filename(file))
         end
       end
 
@@ -145,6 +145,10 @@ module Relaton
           end
           bibdata_instance(doc, xml[:file]) if doc.root.name == "bibdata"
         end.compact
+      end
+
+      def concatenate_and_write_to_files
+        write_to_file(bibcollection.send(output_type))
       end
 
       def split_and_write_to_files
@@ -164,9 +168,16 @@ module Relaton
         end
       end
 
-      def output_type
-        output_format = options[:extension] || "rxl"
-        (output_format == "rxl" ? "to_xml" : "to_#{output_format}").to_sym
+      def output_type(ext=options[:extension])
+        case ext
+        when "rxl", "xml"
+          :to_xml
+        when "yml", "yaml"
+          :to_yaml
+        else
+          puts "The given extension of '#{ext}' is not supported."
+          :to_xml
+        end
       end
 
       def bibdata_instance(document, file)

--- a/spec/acceptance/relaton_concatenate_spec.rb
+++ b/spec/acceptance/relaton_concatenate_spec.rb
@@ -9,7 +9,7 @@ RSpec.describe "Relaton Concatenate" do
       Relaton::Cli.start(command)
 
       expect(Relaton::Cli::RelatonFile).to have_received(:concatenate).
-        with("spec/fixtures", "./tmp/concatenate.rxl", title: "Title")
+        with("spec/fixtures", "./tmp/concatenate.rxl", title: "Title", extension: "rxl")
     end
 
     it "sends concatenate message to the concatenator using new Relaton XML format" do
@@ -19,7 +19,7 @@ RSpec.describe "Relaton Concatenate" do
       Relaton::Cli.start(command)
 
       expect(Relaton::Cli::RelatonFile).to have_received(:concatenate).
-        with("spec/fixturesnew", "./tmp/concatenate.rxl", title: "Title", new: true)
+        with("spec/fixturesnew", "./tmp/concatenate.rxl", title: "Title", new: true, extension: "rxl")
     end
   end
 end

--- a/spec/acceptance/relaton_split_spec.rb
+++ b/spec/acceptance/relaton_split_spec.rb
@@ -19,7 +19,7 @@ RSpec.describe "Relaton Split" do
       Relaton::Cli.start(command)
 
       expect(Relaton::Cli::RelatonFile).to have_received(:split).
-        with("spec/fixtures/sample-collection.xml", "./tmp", new: true)
+        with("spec/fixtures/sample-collection.xml", "./tmp", new: true, extension: "rxl")
     end
   end
 end

--- a/spec/acceptance/relaton_xml2yaml_spec.rb
+++ b/spec/acceptance/relaton_xml2yaml_spec.rb
@@ -10,7 +10,7 @@ RSpec.describe "Relaton xml2yaml" do
         Relaton::Cli.start(command)
 
         expect(Relaton::Cli::XMLConvertor).to have_received(:to_yaml).
-          with("spec/fixtures/sample-collection.xml", { overwrite: false })
+          with("spec/fixtures/sample-collection.xml", overwrite: false, extension: "yaml")
       end
     end
 

--- a/spec/acceptance/relaton_yaml2xml_spec.rb
+++ b/spec/acceptance/relaton_yaml2xml_spec.rb
@@ -10,7 +10,7 @@ RSpec.describe "Relaton yaml2xml" do
         Relaton::Cli.start(command)
 
         expect(Relaton::Cli::YAMLConvertor).to have_received(:to_xml).
-          with("spec/fixtures/sample.yaml", overwrite: false)
+          with("spec/fixtures/sample.yaml", overwrite: false, extension: "rxl")
       end
     end
 

--- a/spec/acceptance/relaton_yaml2xmlnew_spec.rb
+++ b/spec/acceptance/relaton_yaml2xmlnew_spec.rb
@@ -10,7 +10,7 @@ RSpec.describe "Relaton yaml2xmlnew" do
         Relaton::Cli.start(command)
 
         expect(Relaton::Cli::YAMLConvertorNew).to have_received(:to_xml).
-          with("spec/fixturesnew/samplenew.yaml", overwrite: false)
+          with("spec/fixturesnew/samplenew.yaml", overwrite: false, extension: "rxl")
       end
     end
 

--- a/spec/relaton/cli/relaton_file_spec.rb
+++ b/spec/relaton/cli/relaton_file_spec.rb
@@ -23,7 +23,9 @@ RSpec.describe Relaton::Cli::RelatonFile do
     context "with Metanorma XML and different extension" do
       it "extracts XML in output directory with provided extension" do
         Relaton::Cli::RelatonFile.extract(
-          "spec/assets/metanorma-xml", "./tmp/output", extension: "rxml"
+          "spec/assets/metanorma-xml",
+          "./tmp/output",
+          extension: "rxml",
         )
 
         expect(file_exist?("a.rxl")).to be false
@@ -36,7 +38,9 @@ RSpec.describe Relaton::Cli::RelatonFile do
     context "with single Metanorma XML file" do
       it "extracts the XML in the output directory" do
         Relaton::Cli::RelatonFile.extract(
-          "spec/assets/metanorma-xml/a.xml", "./tmp/output"
+          "spec/assets/metanorma-xml/a.xml",
+          "./tmp/output",
+          extension: "rxl",
         )
 
         content = File.read("./tmp/output/a.rxl")
@@ -48,7 +52,9 @@ RSpec.describe Relaton::Cli::RelatonFile do
 
       it "extracts the RFC in the output directory" do
         Relaton::Cli::RelatonFile.extract(
-          "spec/fixtures/draft-celi-acvp-sha-00.xml", "./tmp/output"
+          "spec/fixtures/draft-celi-acvp-sha-00.xml",
+          "./tmp/output",
+          extension: "rxl",
         )
 
         content = File.read("./tmp/output/draft-celi-acvp-sha-00.rxl")
@@ -64,7 +70,9 @@ RSpec.describe Relaton::Cli::RelatonFile do
     context "with YAML & RXL files in source directory" do
       it "combines both type of files into a collection" do
         Relaton::Cli::RelatonFile.concatenate(
-          "spec/fixtures", "./tmp/concatenate.yml"
+          "spec/fixtures",
+          "./tmp/concatenate.yml",
+          extension: "yml",
         )
 
         hashdoc = YAML.load_file("./tmp/concatenate.yml")
@@ -86,6 +94,7 @@ RSpec.describe Relaton::Cli::RelatonFile do
           "./tmp/concatenate.yml",
           title: "collection title",
           organization: "Ribose Inc",
+          extension: "yml",
         )
 
         hashdoc = YAML.load_file("./tmp/concatenate.yml")
@@ -102,6 +111,7 @@ RSpec.describe Relaton::Cli::RelatonFile do
           title: "collection title",
           organization: "Ribose Inc",
           new: true,
+          extension: "yml",
         )
 
         hashdoc = YAML.load_file("./tmp/concatenate.yml")
@@ -120,7 +130,9 @@ RSpec.describe Relaton::Cli::RelatonFile do
         create_fixture_files("sample", file_types)
 
         Relaton::Cli::RelatonFile.concatenate(
-          "spec/fixtures", "./tmp/concatenate.yml"
+          "spec/fixtures",
+          "./tmp/concatenate.yml",
+          extension: "yml",
         )
 
         cleanup_fixture_files("sample", file_types)
@@ -148,7 +160,7 @@ RSpec.describe Relaton::Cli::RelatonFile do
         output_dir = "./tmp/output"
         collection_file = "spec/fixtures/sample-collection.xml"
 
-        Relaton::Cli::RelatonFile.split(collection_file, output_dir)
+        Relaton::Cli::RelatonFile.split(collection_file, output_dir, extension: "rxl")
         content = File.read([output_dir, "cc-34000.rxl"].join("/"))
 
         expect(file_exist?("cc-34000.rxl")).to be true
@@ -161,13 +173,13 @@ RSpec.describe Relaton::Cli::RelatonFile do
         output_dir = "./tmp/output"
         collection_file = "spec/fixturesnew/sample-collection.xml"
 
-        Relaton::Cli::RelatonFile.split(collection_file, output_dir, new: true)
-        content = File.read([output_dir, "cc-34000.rxl"].join("/"))
+        Relaton::Cli::RelatonFile.split(collection_file, output_dir, new: true, extension: "yaml")
+        content = File.read([output_dir, "cc-34000.yaml"].join("/"))
 
-        expect(file_exist?("cc-34000.rxl")).to be true
+        expect(file_exist?("cc-34000.yaml")).to be true
         expect(Dir["#{output_dir}/**"].length).to eq(6)
-        expect(content).to include('<bibdata type="standard">')
-        expect(content).to include("<title>Date and time -- Concepts")
+        expect(content).to include('id: CC34000')
+        expect(content).to include("title: Date and time -- Concepts and vocabulary")
       end
     end
   end

